toolchain/sem_ir/ids.h

```diff
@@ -732,6 +744,14 @@ struct TypeId : public IdBase<TypeId> {
   auto Print(llvm::raw_ostream& out) const -> void;
 };
 
+// The ID of a Clang Source Location.
+struct ClangSourceLocationId : public IdBase<ClangSourceLocationId> {
+  static constexpr llvm::StringLiteral Label = "clang_source_location";
```

Comment:
```suggestion
struct ClangSourceLocId : public IdBase<ClangSourceLocId> {
  static constexpr llvm::StringLiteral Label = "clang_source_loc";
```

While "Location" matches clang, in our code we've been abbreviating it ([per idioms](https://github.com/carbon-language/carbon-lang/blob/trunk/toolchain/docs/idioms.md#abbreviations-used-in-the-code-aka-carbon-abbreviation-decoder-ring)). Maybe this should abbreviate too? (and actually, similar elsewhere you're spelling out "location" for Carbon)

Reply:
Done.